src/_guides/language/analysis-options.md

```diff
@@ -204,6 +204,50 @@ For example, some rules are more appropriate for library packages
 and others are designed for Flutter apps.
 Note that linter rules can have false positives, unlike static analysis.
 
+### Enabling Dart team recommended rules: lints {#lints}
+
+The Dart team provides two sets of recommended linter rules.
+
+Two sets of lints are provided by the package:
+
+Core lints
+: Lints that help identify critical issues that are likely to lead to problems
+when running or consuming Dart code. All code should pass these lints.
+
+Recommended lints
+: Lints that help identify additional issues
+that may lead to problems when running or consuming Dart code,
+and lints that enforce writing Dart using a single, idiomatic style and format.
+All code is encouraged to pass these lints.
+The recommended lints include all the core lints.
+
+To enable one the Dart team's recommended sets of lints,
+add the [lints package]({{site.pub-pkg}}/lints) as a dev dependency:
+
+```terminal
+$ dart pub add --dev lints
+```
+
+Then include the `analysis_options.yaml` file
+corresponding to your preferred set
+in your `analysis_options.yaml` file:
+
+```yaml
+include: package:lints/core.yaml
+```
+
+```yaml
+include: package:lints/recommended.yaml
+```
+
```

Comment:
Consider adding info for Flutter lints?  (Or linking out to the instructions here: https://github.com/flutter/packages/tree/master/packages/flutter_lints ?)

Reply:
I don't think we should have Flutter info here, but we _should_ link out to it. We should probably mention that it builds on `recommended.yaml`. Maybe have a site.alert.tip box near the top of this section?

Reply:
Hmmm... Now I'm rethinking the Flutter lint thing. If most of our readers use Flutter, maybe we _should_ show how to do it. It might still be in a note, though.

@mit-mit what do you think?

Reply:
Perhaps we can have a note above the `### Enabling Dart team recommended rules: lints {#lints}` section that instead points users to https://pub.dev/packages/flutter_lints if they are looking to set up analysis for a Flutter package or application.

Reply:
I took a whack at a note about flutter_lints.